### ISSUE TITLE
Resizable, detachable and floating Structure Panel

### ISSUE THREAD

**Issue by srikat** [soflyy/oxygen-bugs-and-features#289]
It would be of great convenience to have the Structure Panel be detachable, draggable and resizable.

**Comment by royby** [comment]
+1

**Comment by dollhauss** [comment]
Absolutely. Improved drag and drop for moving elements around would also be welcome.

**Comment by demetre19** [comment]
+1

**Comment by muzaffarsiddiqui** [comment]
+1

**Comment by alpabuz** [comment]
We definitely need this. Especially when working on a laptop this is not enough.

**Comment by charlesoharaphoto** [comment]
I second this. For now, the interface is OK, but pretty cramped and rigid. The Structure panel could go even further by allowing opening in a second tab/window. This way it could be put on a second screen, just the way it works with a lot of professional software packages (Photoshop, etc.).

**Comment by webprogrammingsolutions** [comment]
+1

**Comment by atheimer** [comment]
+1 would be so helpful to be able to resize this (or detach etc..)

**Comment by fmommeja** [comment]
+1
